app/test/integration/launch-test.ts

```diff
@@ -32,11 +32,9 @@ describe('App', function(this: any) {
       path: appPath,
       args: [path.join(__dirname, '..', '..', '..', 'out')],
     })
-    return app.start()
-  })
-
-  beforeEach(function() {
```

Comment:
I'm not totally sure I understand this change?

Reply:
That’s an artifact from my use of `eslint-plugin-mocha`. I’ll revert it when I get a chance.